lib/pf/api.pm

```diff
@@ -1348,6 +1348,12 @@ sub handle_accounting_metadata : Public {
             $logger->info("Updating iplog from accounting request");
             $client->notify("update_ip4log", mac => $mac, ip => $RAD_REQUEST{'Framed-IP-Address'}) if ($RAD_REQUEST{'Framed-IP-Address'} );
         }
+        if (pf::util::isenabled($pf::config::Config{advanced}{unreg_on_accounting_stop})) {
+           $client->notify("deregister_node", mac => $mac);
+        }
+    }
+
+        }
```

Comment:
You seem to be adding too many `}`

From what I can tell by adding the 2 `}`, you're shortcuting everything that happens in the rest of the subroutine